src/components/ResultsDisplay/index.jsx

```diff
@@ -0,0 +1,58 @@
+import RouteCard from "../RouteCard/index.jsx";
+import {useEffect, useState} from "react";
+import PlanRouteButton from "../PlanRouteButton/index.jsx";
+import NoRouteCard from "../NoRouteCard/index.jsx";
+import InvalidInputCard from "../InvalidInputCard/index.jsx";
+import {Link, Routes, Route} from "react-router-dom";
+import ClearRouteButton from "../ClearRouteButton/index.jsx";
+
+function ResultsDisplay({selected, destinationSelected}) {
+
+
+    const [journey, setJourney] = useState([])
+    const [boolean, setBoolean] = useState(false)
+    const [status, setStatus] = useState(0)
+
+    let originCode
+    let destinationCode
+
+    if (selected && destinationSelected) {
+        originCode = selected
+        destinationCode = destinationSelected
+    }
+
+    const fetchJourney = async () => {
+        const response = await fetch(`http://localhost:3000/journeys?origin=${originCode}&destination=${destinationCode}`)
+        setStatus(response.status)
+        let journeysArray
```

Comment:
No need for this if it's assigned on the next line: just put `const` at the start of the next line instead